kasa/protocol.py

```diff
@@ -29,14 +29,20 @@ class TPLinkSmartHomeProtocol:
     DEFAULT_TIMEOUT = 5
 
     @staticmethod
-    async def query(host: str, request: Union[str, Dict], retry_count: int = 3) -> Dict:
+    async def query(
+        host: str,
+        request: Union[str, Dict],
+        retry_count: int = 3,
```

Comment:
I have nothing against this change, but would it make more sense to allow passing the wanted port to `__init__`? That would allow us to expose this directly to `SmartDevice` constructors.

Reply:
I think that's better, actually. I'm waiting for feedback to see the #108 use case.